index5.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
+</head>
+<body>
+
+<header class="header">
+    <div class="container">
+        <div class="header__inner">
+            <nav class = "nav">
+                <a class="nav__link" href="index4.html">top players</a>
+                <a class="nav__link" href="index.html">statistics</a>
+                <a class="nav__link" href="index2.html">player activity</a>
+                <a class="nav__link" href="index3.html">player best and worst games</a>
+                <a class="nav__link" href="index5.html">player info</a>
+            </nav>
+        </div>
+    </div>
+</header>
+
+
+
+<div class = "main_win_stat_activity">
+    <section class="win_section">
+        <p>Рейтинг: 2038.67</p>
+        <p>Прогресс за последние игры (12): 31</p>
+    </section>
+    <section class="loose_section">
+        <p>Вы сильнее 90.9% игроков в Bullet</p>
+        <p>Отклонение рейтинга: 45.00</p>
+    </section>
+</div>
+<div class = "main_stat_activity">
+    <section class="win_section">
+        <h2>ПАРТИИ</h2>
+        <p>Всего партий:       1099&emsp;Рейтинговые игры:   1063</p>
+        <p>Турнирных партий:   0&emsp;Партий с берсерком: 0</p>
+        <p>Времени за игрой: 1 день, 8 часов и 19 минут</p>
+    </section>
+    <section class="loose_section">
+        <h2>РЕЗУЛЬТАТИВНОСТЬ</h2>
+        <p>Средний рейтинг соперников	2038.94</p>
+        <p>Побед	526	48%&emsp;Поражений	503	46%</p>
+        <p>Ничьих	70	6%&emsp;Отключений	8	1%</p>
+    </section>
+</div>
+<div class = "main_rate_activity">
+    <section class="win_section">
+        <p>Наивысший рейтинг: 2145&emsp;<time datetime="2022-03-05T00:00:00.000Z">19 февр. 2022 г., 16:44  </time><p>    
+    </section>
+    <section class="loose_section">
+        <p>Наименьший рейтинг: 1912&emsp;<time datetime="2022-03-05T00:00:00.000Z">22 нояб. 2021 г., 13:52  </time><p> 
+    </section>
+</div>
+
+
+
+<footer class="footer">
+        <div class="footer__title">
+            Statistics from Lichess with Love
+        </div>
+        <div class="footer__logo">
+
+        </div>
+</footer>
+
+
+
+</body>
+</html>
```

Comment:
Please, recheck ur layout, avoid float, be consistent with ur values and read about best practices. Good luck!